common/utils/types/payment.ts

```diff
@@ -0,0 +1,4 @@
+export type TPaymentList = {
+  mainTitle: string;
+  mainData: { title: string; data: string[] }[]
+}[];
```

Comment:
these types refer to the PAYMENT component. I think they should be on the same level as the component

If these types will be used somewhere else in other components, then they should be placed at a higher level.

I suggest to put them in this path Payment/types/index.d.ts